docs/api_py/methods/stvas_api.rst

```diff
@@ -5,7 +5,7 @@
 Satellite-transition variable-angle spinning (ST-VAS)
 """""""""""""""""""""""""""""""""""""""""""""""""""""
 
-The following classes are used in simulating satellite-transition variable-angle spinning
+The following classes are used in simulating a satellite-transition variable-angle spinning
```

Comment:
@pjgrandinetti @deepanshs Should the sentence start as 

"The following classes are used in a **simulated** multi-quantum variable-angle spinning spectrum correlating..."

or

"The following classes are used when **simulating a** satellite-transition variable-angle spinning spectrum **which correlates**..."

Reply:
"The following classes are used when simulating a satellite-transition variable-angle spinning spectrum which correlates..."